Makefile.common

```diff
@@ -201,7 +201,7 @@ endif
 .PHONY: common-build
 common-build: promu
 	@echo ">> building binaries"
-	GO111MODULE=$(GO111MODULE) $(PROMU) build --prefix $(PREFIX)
+	GO111MODULE=$(GO111MODULE) $(PROMU) build --prefix $(PREFIX) $(PROMU_BINARIES)
```

Comment:
I'm not sure where these value comes from.

Reply:
This is an optional param used for customizing the build process[0]. This is just a sync up of the upstream prometheus/prometheus Makefile.common.

[0]: https://github.com/prometheus/prometheus/pull/5966